### PR TITLE
Dropbox Update

### PR DESCRIPTION
**Pull request by eepstain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-6762)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by eepstain** [comment]
Requires a developer addition to the integration- Ingestion should be made via the Event Collector and not by a Parsing Rule.

File: Packs/Dropbox/README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
+# Dropbox
+This pack includes Cortex XSIAM content. 
 Collect events from Dropbox's logs.
 This integration was integrated and tested with version 2 of Dropbox API
 
+## Time Zone support for XSIAM
+For supporting Time Zone parsing, time should be set to GMT+00:00;
+1. Open your Dropbox account section.
+2. At the **General**, under **Prefrences**, make sure **Automatic time zone** is set to GMT+00:00 time zone.
```
Comment (ShirleyDenkberg) [inline Packs/Dropbox/README.md:9]: At the General what?  section?  tab?

File: Packs/Dropbox/ReleaseNotes/1_0_20.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Modeling Rules
+
+##### DropboxEventCollector
+
+- Updated the Modeling Rule logic
```
Comment (ShirleyDenkberg) [inline Packs/Dropbox/ReleaseNotes/1_0_20.md:6]:
```suggestion
Updated the Modeling Rule logic
```

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

**Pull request closed without merging**